File: 03 - Online Judges/codechef/decstr.pas
    const
    a='abcdefghijklmnopqrstuvwxyz';
    b='zyxwvutsrqponmlkjihgfedcba';
    var
    T,K,cv,i:byte;
    begin
        Readln(T);
        for cv:=1 to T do
            begin
                Readln(K);
                if K<=25 then begin
                    for i:=K+1 downto 1 do write(a[i]);
                    writeln;
                    end
     
                 else
                if K mod 25 <>0 then
                    begin
                    for i:=(K mod 25)+1 downto 1 do write(a[i]);
                    for i:=K div 25 downto 1 do write(b);
                    writeln;
                    end
     
                else
                begin
                    for i:=K div 25 downto 1 do write(b);
                    writeln;
                end;
             end;
    end. 

File: 03 - Online Judges/codechef/decstr.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
	int n,i,j,k;
	const string a="abcdefghijklmnopqrstuvwxyz",b="zyxwvutsrqponmlkjihgfedcba";
	
	cin>>n;
	for (i=1;i<=n;i++){
		cin>>k;
		if (k<=25){
			for (j=k;j>=0;j--)
				cout<<a[j];
			cout<<endl;
		}
		else if (k%25!=0 && k>25){
			for (j=k%25;j>=0;j--)
				cout<<a[j];
			for (j=k/25;j>=1;j--)
				cout<<b;
			cout<<endl;
		}
		else if (k%25==0){
			for (j=k/25;j>=1;j--)
				cout<<b;
			cout<<endl;
		}
	}
}
